### PR TITLE
enhancement/217 - differentiate color of selected aircraft path from others when projected paths set to "always"

### LINKED ISSUE #217: Differentiate selected aircraft when projected path is always on
When projected path is set to always, it would be good to have the selected vehicle's path change color, to show which path corresponds to the selected vehicle.

### PR DESCRIPTION
**Pull request by cake-pie**
Resolves #217

The purpose of this pull request is to implement different colors for the projected paths of selected vs unselected aircraft when the projected paths option is set to "always".

Selected aircraft use existing colors for both themes; unselected aircraft get a variation with more transparency.  
Let me know if you feel colors need further adjusting.

### CODE REVIEW

**Comment by erikquinn** [comment]
I went to close the issue and PR, as I thought it was saying to make _every_ aircraft have a different color line, so you could tell them apart (to which the answer woulda been "ew, no")... but I see the idea now. Makes sense.

**Comment by erikquinn** [comment]
@cake-pie Just request a review when you're done and ready!

**Comment by cake-pie** [comment]
> make _every_ aircraft have a different color

yikes! :stuck_out_tongue_closed_eyes:

**Review by n8rzz — APPROVED** [review]

File: src/assets/scripts/client/canvas/CanvasController.js [hunk 1]
```diff
@@ -1274,11 +1275,16 @@ export default class CanvasController {
 
         cc.save();
 
-        let strokeStyle = this.theme.RADAR_TARGET.PROJECTION_ARRIVAL;
-
         // future track colors
+        let strokeStyle = this.theme.RADAR_TARGET.PROJECTION_ARRIVAL_ALL;
         if (aircraftModel.category === FLIGHT_CATEGORY.DEPARTURE) {
```
[the remark this answers is not in the document]
Comment (cake-pie) [reply src/assets/scripts/client/canvas/CanvasController.js:1280]: I feel ya, but not sure there's any clever tricks to be pulled here. No matter how you cut it, its two binary tests determining which of four possible values.

**Pull request merged**